CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+## Parity [v1.9.3](https://github.com/paritytech/parity/releases/tag/v1.9.3) (2018-02-21)
```

Comment:
Minor nitpick, this should be 2018-02-20.

Reply:
We are releasing so fast :)